scripts/bazel

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Perform a reproducible build and run tests using Bazel
+
+# Set up Git repository
+mkdir -p bazel
+git clone --depth 1 -b evm-rpc-canister https://github.com/dfinity/ic.git bazel/ic
+cd bazel/ic || exit 1
+test -d .git || exit 1
+git pull
+ln -fs ../../../.. rs/ethereum/evm_rpc
+
+# Temporary until this is merged: https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/19052
+sed -i='' 's/"threadpool":/"tiny-keccak": crate.spec(version = "2.0.2", features = ["keccak"]), "threadpool" :/g' bazel/external_crates.bzl || exit 1
+CARGO_BAZEL_REPIN=1 bazel sync --only=crate_index || exit 1
```

Comment:
Temporary patch equivalent to [!19052](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/19052). Once that MR is merged, we can remove these `sed` and `bazel sync` commands, which will significantly speed up the build process.